torchaudio/datasets/utils.py

```diff
@@ -264,6 +264,25 @@ def walk_files(root, suffix, prefix=False, remove_suffix=False):
                 yield f
 
 
+def get_checksum(dataset, url):
+    """Get checksum of file at the URL corresponding to the Dataset
+
+    Args:
+        dataset (str): Name of dataset . Should match a file name(without extension) in 'datasets/checksums' folder
+        url (str): URL for the file to be downloaded
+    """
+    relative_path = "checksums/" + dataset + ".txt"
+    checksum_file = os.path.join(os.path.dirname(__file__), relative_path)
+    with open(checksum_file) as f:
+        checksum_dict = {k: v for line in f for k, v in [line.strip().split()]}
+        try:
+            checksum = checksum_dict[url]
+        except KeyError:
+            checksum = None
+            pass
+        return checksum
+
+
```

Comment:
Reading such files could be useful, but this particular function wouldn't be generic enough. This also means that new files need to be created to host the checksums.

Since the urls are already hardcoded in each module file, I suggest instead to simply add a hidden dictionary directly in each module along the URLs, e.g.
```python
_CHECKSUMS = {
"http://www.openslr.org/resources/12/dev-clean.tar.gz": "76f87d090650617fca0cac8f88b9416e0ebf80350acb97b343a85fa903728ab3",
"http://www.openslr.org/resources/12/dev-other.tar.gz":  "12661c48e8c3fe1de2c1caa4c3e135193bfb1811584f11f569dd12645aa84365",
...
}
```
and then we can get the checksum by doing
```
                    checksum = _CHECKSUMS.get(url, None)
```

Thoughts?

Reply:
ok will make the suggested changes. I thought it might clutter the module for datasets like commonvoice where we have many urls ,so went for file  approach.